GP/toy_experiments/toy_sin_noise.py

```diff
@@ -57,20 +53,17 @@
         #  Initial noisy data points sampled uniformly at random from the input space.
 
         init_num_samples = 3  # all un-named plots were 33 initial samples
-        X_init = np.random.uniform(0, 10, init_num_samples).reshape(-1, 1)  # sample 7 points at random from the bounds to initialise with
+        X_init = np.random.uniform(0, 10, init_num_samples).reshape(-1,
+                                                                    1)  # sample 7 points at random from the bounds to initialise with
         plot_sample = np.linspace(0, 10, 50).reshape(-1, 1)  # samples for plotting purposes
 
-        Y_init = linear_sin_noise(X_init, noise_coeff, plot_sample, coefficient, modification, fplot=False)
+        Y_init = linear_sin_noise(X_init, noise_coeff, plot_sample, coefficient, modification, fplot=True)
 
         # Initialize samples
         homo_X_sample = X_init.reshape(-1, 1)
         homo_Y_sample = Y_init.reshape(-1, 1)
         het_X_sample = X_init.reshape(-1, 1)
         het_Y_sample = Y_init.reshape(-1, 1)
-        aug_X_sample = X_init.reshape(-1, 1)
```

Comment:
Sorry to do a U-turn on this but perhaps we should keep the Augmented Expected Improvement.